### DISCUSSION TITLE
Is it possible to use a custom DNS?

### DISCUSSION THREAD

**Discussion by ghost** [is-a-dev/register#14817] [Q&A] [answered]
Is it possible to use a custom DNS? Eg. hurricane electric or anything like that?

**Comment by MaskDuck** [comment, marked as answer]
afaik custom DNS require NS records which is 
- not supported yet, but we're planning
- if we actually support then it will be reserved for maintainer or trusted users only

**Reply by ghost** [reply]
Is there any ETA on NS support?

**Reply by wdhdev** [reply]
No
